client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -664,6 +664,177 @@ class WpcomChecklistComponent extends PureComponent {
 			/>
 		);
 	};
+
+	renderAboutTextUpdatedTask = ( TaskComponent, baseProps, task ) => {
+		const { translate } = this.props;
+
+		return (
+			<TaskComponent
+				{ ...baseProps }
+				title={ translate( 'Edit About text' ) }
+				description={ translate(
+					'Update the text we’ve written for you to describe what makes your business unique. ' +
+						'Make your homepage speak to your customers.'
+				) }
+				duration={ translate( '%d minute', '%d minutes', { count: 5, args: [ 5 ] } ) }
+				onDismiss={ this.handleTaskDismiss( task.id ) }
+			/>
+		);
+	};
+
+	renderHomepagePhotoUpdatedTask = ( TaskComponent, baseProps, task ) => {
+		const { translate } = this.props;
+
+		return (
+			<TaskComponent
+				{ ...baseProps }
+				title={ translate( 'Change homepage photo' ) }
+				description={ translate(
+					'Upload your own photoo or choose from a wide selection of free ones to personalize your new site.'
+				) }
+				duration={ translate( '%d minute', '%d minutes', { count: 10, args: [ 10 ] } ) }
+				onDismiss={ this.handleTaskDismiss( task.id ) }
+			/>
+		);
+	};
+
+	renderBusinessHoursAddedTask = ( TaskComponent, baseProps, task ) => {
+		const { translate } = this.props;
+
+		return (
+			<TaskComponent
+				{ ...baseProps }
+				title={ translate( 'Add business hours' ) }
+				description={ translate(
+					'Let your customers know when you’re open or the best time to contact you.'
+				) }
+				duration={ translate( '%d minute', '%d minutes', { count: 8, args: [ 8 ] } ) }
+				onDismiss={ this.handleTaskDismiss( task.id ) }
+			/>
+		);
+	};
+
+	renderServiceListAddedTask = ( TaskComponent, baseProps, task ) => {
+		const { translate } = this.props;
+
+		return (
+			<TaskComponent
+				{ ...baseProps }
+				title={ translate( 'Add a list of services' ) }
+				description={ translate(
+					'Let potential customers and clients know what you have to offer.'
+				) }
+				duration={ translate( '%d minute', '%d minutes', { count: 8, args: [ 8 ] } ) }
+				onDismiss={ this.handleTaskDismiss( task.id ) }
+			/>
+		);
+	};
+
+	renderStaffInfoAddedTask = ( TaskComponent, baseProps, task ) => {
+		const { translate, siteVerticals } = this.props;
+		let staff = translate( 'staff' );
+
+		if ( includes( siteVerticals, 'Health & Medical' ) ) {
+			staff = translate( 'doctors', { context: 'Health & Medical' } );
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.